index.html

```diff
@@ -24,19 +24,18 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
       integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
       crossorigin="anonymous"
-      referrerpolicy="no-referrer"
-    />
+      referrerpolicy="no-referrer">
 
-    <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css">
 
-    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="stylesheet" href="assets/css/style.css">
   </head>
 
-  <body data-spy="scroll" data-target=".navbar" data-offset="40" id="home">
+  <body data-spy="scroll" data-target=".navbar" data-offset="40" id="home-body">
```

Comment:
@Bolaji06 Did you check that nothing breaks on UI with this ID change?

Reply:
Did a good checks on the UI and made sure every part of the UI works as it was before.